### DISCUSSION TITLE
How can I center the globe camera on a specific geo coordinates (phi and theta) ?

### DISCUSSION THREAD

**Discussion by MatteoGauthier** [shuding/cobe#63] [Q&A] [answered]
Hey, I'm trying to find the values of theta and phi to center the globe camera on a specific marker. Does someone know how can I achieve that ?

**Comment by shuding** [comment, marked as answer]
Check this out: https://cobe.vercel.app/docs/showcases/rotate-to-location

**Reply by MatteoGauthier** [reply]
Thanks ! I ended up finding a similar approach

**Comment by MatteoGauthier** [comment]
Code snippet to achieve this : 
```ts
  const locationToAngles = (lat: number, long: number) => {
    return [
      Math.PI - ((long * Math.PI) / 180 - Math.PI / 2),
      (lat * Math.PI) / 180,
    ];
  };

```
